tests/auth_tests/test_forms.py

```diff
@@ -380,21 +380,20 @@ def test_unicode_username(self):
 
     @override_settings(AUTH_USER_MODEL='auth_tests.CustomUserWithLongerUsernameField')
     def test_username_field_max_length_matches_user_model(self):
-        # We authenticate with a username of exactly 255 characters
-        # because we want to test that it is possible to authenticate
-        # with a username that is longer than the 254 default for the form
-        # (for custom user models that allow usernames of that length),
-        # but we cannot test username lengths longer than this due to
+        # Authenticate with a username of exactly 255 characters
+        # to test that it is possible to authenticate with a username
+        # that is longer than the 254 default for the form
+        # (for custom user models that allow usernames of that length).
+        # Username lengths longer than this cannot be tested due to
         # VARCHAR limitations in MySQL
```

Comment:
Sure, I can shorten it. The default username max length is actually 150 characters, but the `AuthenticationForm` has a default max length of 254. I'll change the comment to read:
> MySQL VARCHAR allows up to 255 characters, which is more than max length allowed by AuthenticationForm with the default user model.